great_expectations/expectations/metrics/table_metrics/table_head.py

```diff
@@ -29,29 +34,40 @@ def _pandas(
         execution_engine: PandasExecutionEngine,
         metric_domain_kwargs: dict,
         metric_value_kwargs: dict,
-        metrics: Dict[str, Any],
+        metrics: dict[str, Any],
         runtime_configuration: dict,
-    ):
+    ) -> pd.DataFrame:
         df, _, _ = execution_engine.get_compute_domain(
             metric_domain_kwargs, domain_type=MetricDomainTypes.TABLE
         )
         if metric_value_kwargs.get("fetch_all", cls.default_kwarg_values["fetch_all"]):
             return df
-        return df.head(metric_value_kwargs["n_rows"])
+        n_rows: int = (
+            metric_value_kwargs.get("n_rows")
+            if metric_value_kwargs.get("n_rows") is not None
+            else cls.default_kwarg_values["n_rows"]
+        )
```

Comment:
```suggestion
        n_rows: int = (
            metric_value_kwargs.get("n_rows"), cls.default_kwarg_values["n_rows"])
        )
```
This work?

Reply:
I tried that first, but if `metric_value_kwargs["n_rows"]=None`, it will not use the default.